project-base/storefront/components/Pages/Cart/CartList/CartListItemInfo.tsx

```diff
@@ -1,40 +0,0 @@
-import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
```

Comment:
Im not sure why we removed this isolated component? Wasnt it easier to read?

Reply:
It wasn't easier to work with. Since this was tightly connected with only one place I haven't find it so useful have it in other component. If it include some additional logic I wouldn't touch it - then it would be better readable in separate space. But this way we could split almost everything just because better readability. I personally prefer having the code together because during some styling refactoring it's much easier to work with than switching between different components. Until it would have some additional logic or make a component huge (for which we said like 250 lines).t

But if you don't like the change since I've done refactoring I don't have problem to put it into separated component.